js/aax.js

```diff
@@ -2037,6 +2039,204 @@ module.exports = class aax extends Exchange {
         return result;
     }
 
+    parsePosition (position, market = undefined) {
+        //
+        //    {
+        //        "autoMarginCall": false,
+        //        "avgEntryPrice": "3706.03",
+        //        "bankruptPrice": "2963.3415880000",
+        //        "base": "ETH",
+        //        "code": "FP",
+        //        "commission": "0.02964824",
+        //        "currentQty": "2",
+        //        "funding": "-0.04827355",
+        //        "fundingStatus": null,
+        //        "id": "385839395735639395",
+        //        "leverage": "5",
+        //        "liquidationPrice": "2983.07",
+        //        "marketPrice": "3731.84",
+        //        "openTime": "2021-12-31T18:57:25.930Z",
+        //        "posLeverage": "5.00",
+        //        "posMargin": "14.85376824",
+        //        "quote": "USDT",
+        //        "realisedPnl": "-0.07792179",
+        //        "riskLimit": "10000000",
+        //        "riskyPrice": "3272.25",
+        //        "settleType": "VANILLA",
+        //        "stopLossPrice": "0",
+        //        "stopLossSource": 1,
+        //        "symbol": "ETHUSDTFP",
+        //        "takeProfitPrice": "0",
+        //        "takeProfitSource": 1,
+        //        "unrealisedPnl": "0.51620000",
+        //        "userID": "3829384",
+        //        "ts": 1641027194500
+        //    }
+        //
+        const contract = this.safeString (position, 'symbol');
+        market = this.safeMarket (contract, market);
+        const size = this.safeString (position, 'currentQty');
+        let side = undefined;
+        if (Precise.stringGt (size, '0')) {
+            side = 'buy';
+        } else if (Precise.stringLt (size, '0')) {
+            side = 'sell';
+        }
+        // const maintenanceRate = this.safeString (position, 'maintenance_rate');
+        // const notional = this.safeString (position, 'value');
+        const leverage = this.safeString (position, 'leverage');
+        const unrealisedPnl = this.safeString (position, 'unrealisedPnl');
+        const realisedPnl = this.safeString (position, 'realisedPnl');
+        // Initial Position Margin = ( Position Value / Leverage ) + Close Position Fee
+        const takerFee = this.fees.trading.taker;
```

Comment:
We cannot use the dot notation for accessing dictionary keys, we should use the quoted notation instead: https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#accessing-dictionary-keys

Reply:
`this.fees['trading']['taker']`?

Reply:
Yep, exactly.

Reply:
We have to make sure that we handle spot vs swap fees though.